### PR TITLE
fix(storage-resize-images): ensure content type is always available

### PR DESCRIPTION
**Pull request by Ehesp**
- Fix: Files are not always uploaded with a default content type (e.g. gsutils).
- Docs: File validation is unknown to the user. Added a note which indicates a file must have a content type, and one which starts with the `image/` prefix.

Fixes https://github.com/firebase/extensions/issues/175

### CODE REVIEW

**Comment by rachelsaunders** [comment]
With what GitHub Issue is this PR associated?  Could it please be added to the PR description?

**Comment by Ehesp** [comment]
@rachelsaunders added it in, cross linked it from the issue rather than the PR.

File: storage-resize-images/POSTINSTALL.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ You can test out this extension right away:
 
 1.  Go to your [Storage dashboard](https://console.firebase.google.com/project/${param:PROJECT_ID}/storage).
 
-1.  Upload an image file to the bucket: `${param:IMG_BUCKET}`
+1.  Upload an image file to the bucket: `${param:IMG_BUCKET}`. The file must have a valid [image MIME Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#Image_types), for example `image/png`.
```
Comment (rachelsaunders) [inline storage-resize-images/POSTINSTALL.md]: Is this a really common case that a dev might not have the image MIME type specified?  If not, then I wouldn't put this level of detail here in this uber quickstart section.  Instead, I'd like to explore putting some of these known gotchas within the "Using this extension" section.

Comment (laurenzlong) [reply storage-resize-images/POSTINSTALL.md]: Yes, agreed.

Comment (rachelsaunders) [reply storage-resize-images/POSTINSTALL.md]: Moved this "error" down in the POSTINSTALL file.  Pls double-check that my change and additional text is correct?  Thx!

Comment (rachelsaunders) [reply storage-resize-images/POSTINSTALL.md]: ping on this comment thread :-)

Comment (russellwheatley) [reply storage-resize-images/POSTINSTALL.md]: Looks good to me.

Comment (rachelsaunders) [reply storage-resize-images/POSTINSTALL.md]: @laurenzlong Are you ok with the changes?

**Pull request merged**